core/src/backends/lnd/rest/node.rs

```diff
@@ -96,4 +96,16 @@ impl NodeMethods for LndRest {
 
         Ok(data.try_into()?)
     }
+
+    async fn get_invoice(&self, payment_hash: String) -> Result<Invoice, Error> {
+        let url = format!("{0}/v1/invoice/{1}", self.config.url, payment_hash);
```

Comment:
No need to add indexes if you don't reuse values.
```suggestion
        let url = format!("{}/v1/invoice/{}", self.config.url, payment_hash);
```